common/src/main/java/org/apache/uniffle/common/rpc/MonitoringServerTransportFilter.java

```diff
@@ -35,12 +35,12 @@ public MonitoringServerTransportFilter(GRPCMetrics grpcMetrics) {
   }
 
   public Attributes transportReady(Attributes transportAttrs) {
-    grpcMetrics.setGauge(GRPC_SERVER_CONNECTION_NUMBER_KEY, connectionSize.incrementAndGet());
+    grpcMetrics.incGauge(GRPC_SERVER_CONNECTION_NUMBER_KEY);
```

Comment:
Should we remove the variable `connectionSize`?

Reply:
Yes